web/src/components/Header/index.tsx

```diff
@@ -24,17 +23,27 @@ const StyledContainer = styled(Container)`
   }
 `
 
-const menuItems = ['about', 'upload', 'maps']
+const menuItems = [
+  { text: 'Visit WOEIP', route: 'about' },
+  { text: 'Add Data', route: 'upload' },
+  { text: 'View Maps', route: 'maps' }
+]
 
-const Header: React.FunctionComponent = () => {
-  const [activeItem, setActiveItem] = useState<MenuItemProps['name']>(
-    menuItems[menuItems.length - 1]
+interface TParams {
```

Comment:
I would suggest to keep your interface names to be more specific to reflect its use case. So this can be renamed to `RouteParams` or `RouteProps`. Avoid things like `TParams` or `IParams`